### PR TITLE
Merge Media Editing branch into develop

### PR DESCRIPTION
**Pull request by leandroalonso**
Adds Media Editing capabilities.

### To test

1. In Aztec, tap an image
2. Tap "Edit"
3. Make sure the Media Editor opens

All the other changes were tested in the PRs submitted to this feature branch.

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: Podfile [hunk 1]
```diff
@@ -145,7 +145,7 @@ target 'WordPress' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :commit => '7963eeed7aaa0ca9529fcbaf7408ca12fe132122'
+    gutenberg :commit => '1616ba1b8f5c02cf4f928410a92ce65e2e594f37'
```
Comment (diegoreymendez) [inline Podfile]: Is this change intentional?

File: Rakefile [hunk 2]
```diff
@@ -126,12 +126,12 @@ namespace :assets do
     next unless Dir['WordPress/Resources/AppImages.xcassets/AppIcon-Internal.appiconset/*.png'].empty?
     Dir.mktmpdir do |tmpdir|
       puts "Generate internal icon set"
-      if system("export PROJECT_DIR=#{Dir.pwd}/WordPress && export TEMP_DIR=#{tmpdir} && ./Scripts/BuildPhases/AddVersionToIcons.sh >/dev/null 2>&1") != 0
```
Comment (diegoreymendez) [inline Rakefile]: You should probably roll back the changes to this file.

File: Podfile (hunk 1 above)
Comment (leandroalonso) [reply Podfile]: Yes, it is!
